rq_gevent_worker.py

```diff
@@ -10,7 +10,7 @@
 import gevent
 import gevent.pool
 from rq import Worker
-from rq.job import Status
+from rq.job import JobStatus
```

Comment:
We need to compatible with old rq, check rq version to decide importing Status or JobStatus.